### PR TITLE
Show nav when a user scrolls up on a small screen

### PR DESCRIPTION
**Pull request by hanbyul-here**
part of https://github.com/NASA-IMPACT/delta-ui/issues/155

the code needs to be cleaned up better, but I think @ricardoduplos @aboydnw can check how it behaves and decides if this is the direction we want to take.

### CODE REVIEW

**Comment by aboydnw** [comment]
Is there a reason we only have it for small screens right now? 
also, the important part of the nav here would the items in the hamburger menu and that doesn't seem to be appearing until the user scrolls all the way up.

**Comment by hanbyul-here** [comment]
huh I just assumed it is for small screens since I've observed this behavior on mobile more often? we can implement it for large screens too. missing 🍔 was my bad. just pushed a fix.

File: app/scripts/components/common/page-header.js [hunk 1]
```diff
@@ -359,19 +360,38 @@ function PageHeader() {
...
-    }
   }, [isLargeDown]);
 
   const closeNavOnClick = useCallback(() => setGlobalNavRevealed(false), []);
 
+  const [scrollDir, scrolledAmount] = useScrollDirection();
+  const [navTopStyle, setNavTopStyle] = useState({ position: 'relative' });
+
+  useEffect(() => {
+    const navHeight = isLargeDown ? (isMediumDown ? 64 : 76) : 88;
```
[the remark this answers is not in the document]
Comment (hanbyul-here) [reply app/scripts/components/common/page-header.js:371]: Daniel, I understand your conecern, but this pr is not ready for review yet. I mainly wanted to confirm with anthony and Ricardo  if it is worth pushing this direction.

[the remark this answers is not in the document]
Comment (danielfdsilva) [reply app/scripts/components/common/page-header.js:371]: I didn't want to sound pushy, apologies. Sometimes refactoring things like this changes the needed approach a bit and it is easier if caught early on.

In the future I'll wait until prompted before doing any type of review. I'd suggest that we start using the Draft Pr feature for these cases as well.
And again, sorry for the hastiness.

[the remark this answers is not in the document]
Comment (hanbyul-here) [reply app/scripts/components/common/page-header.js:371]: Ah I somehow thought draft PR doesn't generate Preview so I went with an actual PR. but I just tested (https://github.com/NASA-IMPACT/delta-ui/pull/207) and I was mistaken. (not related but I learned a hot tip that Netlify will skip the build if a commit message includes `[skip ci]`)

**Comment by hanbyul-here** [comment]
closing for https://github.com/NASA-IMPACT/delta-ui/pull/207

**Pull request closed without merging**